Backend/Controllers/LevelController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Backend.Logic;
+using Backend.Models.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Controllers
+{
+    [ApiController]
+    [Route("Levels")]
+    public class LevelController : Controller
+    {
+        private readonly LevelLogic _logic;
+        private readonly UserLevelLogic _userLevelLogic;
+        public LevelController(LevelLogic logic, UserLevelLogic userLevelLogic)
```

Comment:
"Current practice is not to use underscores at all. Disambiguation between private member variables and parameters with the same name should done using 'this.'. In fact all references to private members should be prefixed with 'this.'."